storage/aws/mq/versions.tf

```
@@ -3,11 +3,11 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 5.3.0"
+      version = ">= 5.4.0"
     }
     random = {
       source  = "hashicorp/random"
-      version = ">= 3.4.3"
+      version = "~> 3.5.1"
```

The recommended way is to let precise version binding decision to the root module.

```suggestion
      version = "=> 3.5.1"
```

---

done